src/test/lib/precomp.h

```diff
@@ -35,6 +35,11 @@
 #define WIN_ASSERT CXPLAT_FRE_ASSERT
 #endif
 #include "karray.h"
+
+#ifndef GLOBAL_FOR_KERNEL
+#define GLOBAL_FOR_KERNEL
+bool UseQTIP = false;
+#endif
```

Comment:
Is this necessary any more?

Reply:
Each test has `extern bool UseQTIP` which seems to be visible by user mode for kernel test. However kernel mode cannot link quic_gtest.cpp which has `bool UseQTIP = false`.

Reply:
Just an idea: I think the ideal solution is passing this flag down from the gtest program along with those test ioctls. It should be a trivial change.